src/ripple/rpc/impl/ServerHandlerImp.cpp

```diff
@@ -738,24 +728,23 @@ ServerHandlerImp::statusResponse(
     std::string reason;
     if (app_.serverOkay(reason))
     {
-        msg.status = 200;
-        msg.reason = "OK";
+        msg.result(beast::http::status::ok);
         msg.body = "<!DOCTYPE html><html><head><title>" + systemName() +
             " Test page for rippled</title></head><body><h1>" +
                 systemName() + " Test</h1><p>This page shows rippled http(s) "
                     "connectivity is working.</p></body></html>";
     }
     else
     {
-        msg.status = 500;
-        msg.reason = "Internal Server Error";
+        msg.result(beast::http::status::internal_server_error);
         msg.body = "<HTML><BODY>Server cannot accept clients: " +
             reason + "</BODY></HTML>";
     }
     msg.version = request.version;
-    msg.fields.insert("Server", BuildInfo::getFullVersionString());
-    msg.fields.insert("Content-Type", "text/html");
-    prepare(msg, beast::http::connection::close);
+    msg.insert("Server", BuildInfo::getFullVersionString());
```

Comment:
looks like beast now has an enum for standard header fields (convertible to string) - should we consider switching to the enum in places where we build requests or responses like this?